ft_defaults.m

```diff
@@ -244,127 +244,73 @@
-    if ft_platform_supports('matlabversion', -inf, '2021a'), ft_hastoolbox('compat/matlablt2021b', 3, 1); end
-    if ft_platform_supports('matlabversion', -inf, '2021b'), ft_hastoolbox('compat/matlablt2022a', 3, 1); end
-    if ft_platform_supports('matlabversion', -inf, '2022a'), ft_hastoolbox('compat/matlablt2022b', 3, 1); end
-    if ft_platform_supports('matlabversion', -inf, '2022b'), ft_hastoolbox('compat/matlablt2023a', 3, 1); end
-    if ft_platform_supports('matlabversion', -inf, '2023a'), ft_hastoolbox('compat/matlablt2023b', 3, 1); end
-    if ft_platform_supports('matlabversion', -inf, '2023b'), ft_hastoolbox('compat/matlablt2024a', 3, 1); end
-    if ft_platform_supports('matlabversion', -inf, '2024a'), ft_hastoolbox('compat/matlablt2024b', 3, 1); end
-    if ft_platform_supports('matlabversion', -inf, '2024b'), ft_hastoolbox('compat/matlablt2025a', 3, 1); end
-    % this deals with compatibility with all OCTAVE versions
-    if ft_platform_supports('octaveversion', -inf, +inf),    ft_hastoolbox('compat/octave', 3, 1); end
-  end
-  
-  try
-    % these contains template layouts, neighbour structures, MRIs and cortical meshes
-    ft_hastoolbox('template/layout',      1, 1);
-    ft_hastoolbox('template/anatomy',     1, 1);
-    ft_hastoolbox('template/headmodel',   1, 1);
-    ft_hastoolbox('template/electrode',   1, 1);
-    ft_hastoolbox('template/neighbours',  1, 1);
-    ft_hastoolbox('template/sourcemodel', 1, 1);
-  end
-  
-  try
-    % this is used in ft_statistics
-    ft_hastoolbox('statfun', 1, 1);
-  end
-  
-  try
-    % this is used in ft_definetrial
-    ft_hastoolbox('trialfun', 1, 1);
-  end
-  
-  try
-    % this contains the low-level reading functions
-    ft_hastoolbox('fileio', 1, 1);
-  end
-  
-  try
-    % this is for filtering etc. on time-series data
-    ft_hastoolbox('preproc', 1, 1);
-  end
-  
-  try
-    % this contains forward models for the EEG and MEG volume conductor
-    ft_hastoolbox('forward', 1, 1);
-  end
-  
-  try
-    % this contains inverse source estimation methods
-    ft_hastoolbox('inverse', 1, 1);
-  end
-  
-  try
-    % this contains intermediate-level plotting functions, e.g. multiplots and 3-d objects
-    ft_hastoolbox('plotting', 1, 1);
-  end
-  
-  try
-    % this contains intermediate-level functions for spectral analysis
-    ft_hastoolbox('specest', 1, 1);
-  end
-  
-  try
-    % this contains the functions to compute connectivity metrics
-    ft_hastoolbox('connectivity', 1, 1);
-  end
-  
-  try
-    % this contains test scripts
-    ft_hastoolbox('test', 1, 1);
+    ft_hastoolbox(addtopath, 3, 1);
   end
   
-  try
-    % this contains the functions for spike and spike-field analysis
-    ft_hastoolbox('contrib/spike', 1, 1);
-  end
+  addtopath = {'template/layout'     'template/anatomy' 'template/headmodel' 'template/electrode' 'template/neighbours' 'template/sourcemodel'}; % these contains template layouts, neighbour structures, MRIs and cortical meshes
```

Comment:
I shortly worried about the fieldtrip-lite zip file on the ftp, which excludes the template files. But I checked: the directories are still there, so that should not pose a problem.